CHANGELOG.md

```diff
@@ -1,49 +1,74 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
-## [0.4.2]
+## [0.5.0]
+
 ### Added
-- fix: local.table is object with no attributes on destroy
+
+- Add support for terraform `v0.15`
+- Add support for terraform `v0.14`
+## [0.4.2]
+
+### Fixed
+
+- Fix `local.table is object with no attributes on destroy`
```

Comment:
just thinking out loud here... can we release 2 versions.... 0.5.0 with 0.14 but mark it as unreleased and 0.6.0 with v0.15 support....

in case we need to fix 0.14 only issues we could use the 0.5.0 line in this case...

wdyt? @soerenmartius

Reply:
Let's keep it simple and just release once version

Reply:
but we have it for others that we have a shadow release at least for 0.14... please increade 0.5.0 () for 0.14 and 0.6.0 for